package.json

```diff
@@ -105,7 +105,8 @@
     "sql-summary": "^1.0.1",
     "stackman": "^4.0.1",
     "traceparent": "^1.0.0",
-    "unicode-byte-truncate": "^1.0.0"
+    "unicode-byte-truncate": "^1.0.0",
+    "url-parse": "^1.4.7"
```

Comment:
removed with using the default parser.

Reply:
Fixed in https://github.com/elastic/apm-agent-nodejs/pull/1792/commits/38fdf0b00bc64ff5933f58c194a74a4b54765f88